Source/Core/DolphinQt/Config/Graphics/EnhancementsWidget.cpp

```diff
@@ -282,9 +282,9 @@ void EnhancementsWidget::SaveSettings()
 void EnhancementsWidget::AddDescriptions()
 {
   static const char TR_INTERNAL_RESOLUTION_DESCRIPTION[] =
-      QT_TR_NOOP("Controls the rendering resolution. A high resolution greatly improves visual "
+      QT_TR_NOOP("Controls the AI upscaling goodness. A high resolution greatly improves visual "
```

Comment:
Does not adequately describe the advanced algorithms used to achieve these marvellous results.

Reply:
Yeah we'll have to add a scroll box so we can fit the entire whitepaper in.

Reply:
We should look into adding a PDF renderer into Externals: https://wiki.qt.io/Handling_PDF#Rendering